src/resources/views/filters/text.blade.php

```diff
@@ -0,0 +1,102 @@
+{{-- Text Backpack CRUD filter --}}
+
+<li filter-name="{{ $filter->name }}"
+	filter-type="{{ $filter->type }}"
+	class="dropdown {{ Request::get($filter->name)?'active':'' }}">
+	<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{{ $filter->label }} <span class="caret"></span></a>
+	<div class="dropdown-menu">
+		<div class="form-group backpack-filter m-b-0">
+			<div class="input-group">
+		        <input class="form-control pull-right"
+		        		id="text-filter-{{ str_slug($filter->name) }}"
+		        		type="text"
+						@if ($filter->currentValue)
+							value="{{ $filter->currentValue }}"
+						@endif
+		        		>
+		        <div class="input-group-addon">
+		          <a class="text-filter-{{ str_slug($filter->name) }}-clear-button" href=""><i class="fa fa-times"></i></a>
+		        </div>
+		    </div>
+		</div>
+	</div>
+</li>
+
+{{-- ########################################### --}}
+{{-- Extra CSS and JS for this particular filter --}}
+
+{{-- FILTERS EXTRA CSS  --}}
+{{-- push things in the after_styles section --}}
+
+@push('crud_list_styles')
+    {{-- <link href="{{ asset('vendor/adminlte/plugins/datepicker/datepicker3.css') }}" rel="stylesheet" type="text/css" /> --}}
+	<style>
+		.input-group.date {
+			width: 320px;
+			max-width: 100%;
+		}
+	</style>
+@endpush
+
+
+{{-- FILTERS EXTRA JS --}}
+{{-- push things in the after_scripts section --}}
+
+@push('crud_list_scripts')
+	<!-- include select2 js-->
+	{{-- <script type="text/javascript" src="{{ asset('vendor/adminlte/plugins/datepicker/bootstrap-datepicker.js') }}"></script> --}}
+  <script>
+		jQuery(document).ready(function($) {
+			$('#text-filter-{{ str_slug($filter->name) }}').on('change', function(e) {
+
+				var parameter = '{{ $filter->name }}';
+				var value = $(this).val();
+
+				@if (!$crud->ajaxTable())
```

Comment:
Personally I'd remove the `!` but it's neither here nor there (why talk in negatives when we can be positive)?